staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -3387,6 +3413,16 @@ type TopologySpreadConstraint struct {
 	// in their corresponding topology domain.
 	// +optional
 	LabelSelector *metav1.LabelSelector `json:"labelSelector,omitempty" protobuf:"bytes,4,opt,name=labelSelector"`
+	// NodeInclusionPolicies instructs how we treat nodeAffinity/nodeTaint policy
+	// when calculating pod topology spread skew.
+	// - NodeAffinity(nodeAffinity/nodeSelector):
+	//   - Respect (default): nodes matching nodeAffinity/nodeSelector will be included.
+	//   - Ignore: all nodes will be included.
+	// - NodeTaint(node taints):
+	//   - Respect: tainted nodes that tolerate the incoming pod, along with regular nodes, will be included.
+	//   - Ignore (default): all nodes will be included.
+	// +optional
```

Comment:
add a comment indicating it's an alpha feature. (check other alpha fields for reference)